### DISCUSSION TITLE
Hide header/title

### DISCUSSION THREAD

**Discussion by CaptainBetty** [makeusabrew/bootbox#848] [Ideas]
Would love an easy way to hide the header/title area of the modal without loosing the close button. Without supplying a Title, there's just a lot of empty space and being able to hide the header area would be a welcome option.

**Comment by tarlepp** [comment]
Does bootstrap support that?

**Comment by CaptainBetty** [comment]
Not that I can find in the documentation. They changed how modals worked
from v4.5 to v5.3 so I'm trying to find a workaround short of manipulating
it myself.

On Fri, Jan 12, 2024 at 3:01 PM Tarmo Leppänen ***@***.***>
wrote:

> Does bootstrap support that?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/bootboxjs/bootbox/discussions/848#discussioncomment-8114271>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAW62RJUL5OABVO5BGWUBR3YOGQCDAVCNFSM6AAAAABBYWLETGVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DCMJUGI3TC>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
*Jon-Paul LeClair*

**Reply by CaptainBetty** [reply]
Best I can do is to use the custom dialog and disable the close button; this turns of the header section completely. Since the dialogs will have buttons to interact with (OK / NO/ Cancel, etc...) I don't think this is big deal. It would be nice however for Bootstrap (and or Bootbox) to handle an empty title a bit more gracefully while allowing a close button. This workaround should work.

**Reply by tarlepp** [reply]
You should raise an issue to Bootstrap repository about this - Imho addind that kind of feature is not something that should be implemented to Bootbox directly.

**Comment by tiesont** [comment]
Bootbox 4 and below would more or less generate the look you're after - we used to omit the header markup and append the close button to the body instead when the `title` option was blank. Unfortunately, that doesn't work out-of-the-box with Bootstrap 5 - the event handling depends on the modal having the structure defined by the Bootstrap team.

**Reply by CaptainBetty** [reply]
Thanks. I am aware. My issue was migrating from Bootstrap 4 to 5. 

I'm mitigating the issue by disabling the close button and relying on the action buttons. This will suffice for my application.
